druid-shell/src/window.rs

```diff
@@ -35,6 +35,14 @@ impl Deref for WindowHandle {
     }
 }
 
+/// A context supplied to most `WinHandler` methods.
+pub trait WinCtx {
```

Comment:
What is the relationship between this and `WindowHandle`?

Reply:
There's a bunch of overlap, but the difference is ownership - `WindowHandle` has shared ownership and the `WinCtx` is mutable access to state singly owned by the platform window handler. I'd like to basically migrate as much as possible to the latter, possibly even get rid of the `connect` method.